source/manual/alerts/email-alerts-travel-medical.html.md

```diff
@@ -5,29 +5,25 @@ section: Icinga alerts
 subsection: Email alerts
 layout: manual_layout
 parent: "/manual.html"
-last_reviewed_on: 2020-07-16
+last_reviewed_on: 2020-08-18
 review_in: 6 months
 ---
 
-There is [a check][email-check] to verify that emails are sent for
-[drug and medical device alerts][] and [travel advice updates][]. These checks
-are run via Jenkins: [Drug and medical device alerts check][drug-alerts-check]
-and [Travel advice alerts check][travel-advice-check].
+There is [a check][email check] to verify that emails are sent for [medical
+safety alerts] and [travel advice updates]. These checks are run via Jenkins:
+[Medical safety alerts check][medical safety check] and [Travel advice
+alerts check][travel advice check].
```

Comment:
It might be useful to add a prefix to this documentation about what these actually test?

For example:

```
We expect that when new GOV.UK content is published, or when existing content is updated in a significant way, emails will be sent to any subscribers to notify them of the change. We actively monitor this for [medical safety alerts] and [travel advice updates] with the [Medical safety alerts check][medical safety check] and [Travel advice
and [Travel advice alerts check][travel-advice-check]. These checks determine a list of content that we expect subscribers to have been emailed for, and then enquire that a Gmail account which we have subscribed to these alerts, has received the email. We do this by using the Gmail API to query which emails the account has received. The checks fail if the Gmail account has not received an expected email within a sufficient time period.

```

Reply:
Good idea, I've slimmed down your suggestion and rejigged it a bit but I think it gets across what these checks are actually doing.